### ISSUE TITLE
LA-Map access from the landing page on pct.bike

### ISSUE THREAD

**Issue by usr110** [npct/pct-shiny#671]
Previously when the [map of local-authorities](https://pct.bike/la-map.html) was launched, it was mentioned in the `what's-new` section. I believe we need a better and permanent solution to make it available. I am thinking of adding it as a link in the top menu. What do you think? Any suggestions? 
@npct/admin

**Comment by Robinlovelace** [comment]
I agree: a link to the local authority map would be very useful for people wanting to get a quick overview of the spatial distribution of cycling and short trips that could be replaced by cycling. Great idea!

**Comment by Robinlovelace** [comment]
Any update on this, are you up for giving it a go @usr110 ?
